### PR TITLE
docs: expose default CSC configurations in the CSC operational guide section

### PR DESCRIPTION
**Pull request by Ebaneck**
**Component**:

'docs', 'csc'

**Context**:

see #2522  

**Summary**:

- Expose default CSC configs as specified in salt/metalks/addons/<name>/config/ directory
  within docs.
- Fix Static user password change procedure

**Acceptance criteria**: 

---

Closes: #2522

### CODE REVIEW

File: docs/operation/cluster_and_service_configuration.rst [hunk 1]
```diff
@@ -10,9 +10,58 @@ Managing Cluster and Services Configurations
 
 Newly deployed **MetalK8s** clusters come with chosen default values for most
 Cluster services. These default values are transparent to Admin users
-and can be customized at any point in time given that Admins follow the
+and can be customized at any point in time given that Administrators follow the
 documented procedure to the later.
 
+Managing default runtime Service Configurations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+MetalK8s addons (Alertmanager, Dex, Grafana and Prometheus) ships with default
```
Comment (alexandre-allard-scality) [inline docs/operation/cluster_and_service_configuration.rst:18]:
```suggestion
MetalK8s addons (Alertmanager, Dex, Grafana and Prometheus) are shipped with default
```

File: docs/operation/cluster_and_service_configuration.rst [hunk 2]
```diff
@@ -10,9 +10,58 @@ Managing Cluster and Services Configurations
...
+deployed in a MetalK8s cluster:
+
+Alertmanager
+""""""""""""
+The following base Alertmanager configurations are required for starting up
+Alertmanager within a MetalK8s cluster.
+To configure Alertmanager, override the default values of alertmanager.yaml
+file as specified below in it's corresponding Cluster and Service ConfigMap
+(metalk8s-alertmanager-config).
+This document below describes parameters that you can set.
```
Comment (alexandre-allard) [inline docs/operation/cluster_and_service_configuration.rst]: It's not a list of parameters that can be set, but just a default basic configuration, user could add some configuration that are not described in this file, at least for Alertmanager (I don't know if it's true for Dex).

**Review by alexandre-allard — APPROVED** [review]

**Review by alexandre-allard — APPROVED** [review]

**Pull request merged**